jdi-light-html-tests/src/test/java/io/github/epam/exapmle/common/CheckboxExampleTests.java

```diff
@@ -0,0 +1,56 @@
+package io.github.epam.exapmle.common;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.html5Page;
+import static io.github.com.pages.HtmlElementsPage.acceptConditions;
+import static io.github.epam.html.tests.elements.BaseValidations.baseValidation;
+import static io.github.epam.html.tests.site.steps.States.shouldBeLoggedIn;
+
+public class CheckboxExampleTests implements TestsInit {
+
+    String labelText = "Accept terms and conditions";
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        html5Page.shouldBeOpened();
+        acceptConditions.check();
+    }
+
+    @Test
+    public void getLabelTextTest() {
+        acceptConditions.label().is().text(labelText);
+    }
+
+    @Test
+    public void checkTest() {
+        acceptConditions.check();
+        acceptConditions.is().selected();
+    }
+
+    @Test
+    public void uncheckTest() {
+        acceptConditions.uncheck();
+        acceptConditions.is().deselected();
+    }
+
+    @Test
+    public void clickTest() {
```

Comment:
Seems like this test doesn't check anything. It would be good if you rewrote it.